.github/workflows/CI.yml

```diff
@@ -32,14 +49,10 @@ jobs:
 
       - name: Run conda build
         shell: bash -el {0}
-        # For the build, these environment variables would usually be set based on repo tags,
-        # but they can be these defaults instead since we don't use the build
-        env:
-          GIT_DESCRIBE_TAG: test
-          GIT_DESCRIBE_NUMBER: 0
         run: conda build . --no-anaconda-upload
```

Comment:
It would be nice if we could [use the same action without uploading to be certain we were replicating exactly what it was doing](https://github.com/uibcdf/action-build-and-upload-conda-packages/issues/2).

Yes I've [looked at the source code](https://github.com/uibcdf/action-build-and-upload-conda-packages/blob/main/action.yml#L121-L126).

There is no action required, I am just documenting stuff for later.

Reply:
Yeah, I had the same thought. Unfortunately the action we're using doesn't have a "no-upload" option. Possibly worth trying to get one added?

Reply:
Ooops, just followed the link to the issue you opened - nice!